javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,256 @@
+
+const questionElement = document.getElementById("question");
+const optionButton = document.getElementById("option-buttons");
+const nextButton = document.getElementById("next-btn");
+const questionContainerElement = document.getElementById("question");
+const exitButton = document.getElementById("exit-btn");
+const startButton = document.getElementById("start-btn");
+const restartButton = document.getElementById("restart-btn");
+
+
+
+//start from 0//
+let currentQuestionIndex = 0;
+let score = 0;
+
+const playersInfo = [
+    {
+        name: "Player1",
+        score: 0,
+    },
+    {
+        name: "Player2",
+        score: 0,
+    },
+];
```

Comment:
I see that you have the right idea! 💪  But you did not actually continue to implement what the exercise asked of you. Using the two inputs where the user inputs the names of the two players, you need to show four buttons: two Correct buttons and two Wrong buttons. When clicked, those buttons will increase the scores of the players. When a player scores 10 points, they win!

Reply:
It might be confusing that this is not part of the actual quiz yet. We do not ask that the players get their score based on the quiz answers. Right now we just want to see you implement two counters, which you control with two buttons each. Simple :)